plugins/demoplugin.c

```diff
@@ -207,8 +207,6 @@ static void demo_cleanup(GeanyPlugin *plugin, gpointer data)
 
 void geany_load_module(GeanyPlugin *plugin, GModule *module, gint geany_api_version)
 {
-	/* main_locale_init() must be called for your package before any localization can be done */
-	main_locale_init(LOCALEDIR, GETTEXT_PACKAGE);
```

Comment:
will this work?  there are `_()` calls before the `main_locale_init()` ones, so they probably won't work (well, I write "probably", but I believe "definitely")